GNUmakefile

```diff
@@ -202,25 +200,22 @@ cstor-volume-mgmt:
 	@echo "----------------------------"
 	@PNAME="cstor-volume-mgmt" CTLNAME=${VOLUME_MGMT} sh -c "'$(PWD)/buildscripts/build.sh'"
 
-#Use this to build cstor-volume-grpc
 cstor-volume-grpc:
```

Comment:
@princerachit - can this be completely removed from the Makefile since this gets built in volume-mgmt?

Reply:
@princerachit this needs to be renamed such that it reflects code generation

Reply:
The issue here is that travis does not have protoc installed. So we need to run this target locally to generate .pb.go file. and then target cstor-volume-mgmt.
About renaming I will update.